services/scabbard/libscabbard/src/store/scabbard_store/diesel/mod.rs

```diff
@@ -14,7 +14,7 @@
 
 mod models;
 mod operations;
-mod schema;
+pub(crate) mod schema;
```

Comment:
This can be scoped to this diesel module and it's children

Reply:
fixed